src/index.js

```diff
@@ -5,8 +5,8 @@ const fp = require("fastify-plugin");
  * by setting the relevant response headers.
  * @param {object} server - Fastify instance.
  */
-async function plugin(server) {
-	server.addHook("onRequest", async (req, res) => {
+async function fastifyDisablecache(server) {
+	server.addHook("onRequest", async (_req, res) => {
```

Comment:
non necessary change, basically _ indicates parameters not used. But not relevant to this PR.